backend/consensus/base.py

```
@@ -301,6 +302,10 @@ def contract_snapshot_factory(
     Returns:
         ContractSnapshot: A new ContractSnapshot instance.
     """
+    try:
+        contract_address = to_checksum_address(contract_address)
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)